src/index.html

```diff
@@ -16,7 +16,7 @@
         {% endfor %}
       </script>
 
-      <title>Viewer NG</title>
+      <title>iViewer</title>
```

Comment:
I think that's a matter of taste. I don't have a strong feeling toward one or the other. To be succinct in the title is perhaps preferable, especially if image names are long. After all, as you said, it's just within the app (for display) and has no technical implications.